index.js

```diff
@@ -112,11 +112,9 @@ async function run() {
 
   // Determine Result
   passed = true
-  var non_signers = []
   for (const i in commit_authors) {
     if (commit_authors[i]['signed'] == false) {
       passed = false;
-      non_signers.push(i)
```

Comment:
remove it completely as we were anyway breaking after the first case

also, list of users who have not signed is reported on line 101